_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -439,77 +464,69 @@ that are not covered by the automated scripts. If you wish to update further, fi
-Guide]
-* link:refarch2-guides/eks-migration-guides/README.md[EKS Service Migration Guide]
-* link:refarch2-guides/iam-cross-account.md[iam-cross-account Service Migration Guide]
-* link:refarch2-guides/iam-groups.md[iam-groups Service Migration Guide]
-* link:refarch2-guides/iam-user-password-policy.md[iam-user-password-policy Service
-Migratirefarch2-guides/on Guide]
-* link:refarch2-guides/jenkins.md[Jenkins Service Migration Guide]
-* link:refarch2-guides/kms-master-key.md[kms-master-key Service Migration Guide]
-* link:refarch2-guides/memcached.md[Memcached Service Migration Guide]
-* link:refarch2-guides/openvpn-server.md[OpenVPN Server Service Migration Guide]
-* link:refarch2-guides/rds.md[RDS Service migration Guide]
-* link:refarch2-guides/redis.md[Redis Service Migration Guide]
-* link:refarch2-guides/route53-private.md[Route 53 (private) Migration Guide]
-* link:refarch2-guides/route53-public.md[Route 53 (public) Migration Guide]
-* link:refarch2-guides/sns-topics.md[sns-topics Service Migration Guide]
-* link:refarch2-guides/vpc.md[VPC (app) Migration Guide]
-* link:refarch2-guides/vpc-mgmt.md[VPC (mgmt) Migration Guide]
+* link:./refarch2-guides/alb.adoc[ALB Service Migration Guide]
+* link:./refarch2-guides/asg.adoc[ASG Service Migration Guide]
+* link:./refarch2-guides/aurora.adoc[Aurora Service Migration Guide]
+* link:./refarch2-guides/cloudtrail.adoc[CloudTrail Service Migration Guide]
+* link:./refarch2-guides/cloudwatch-dashboard.adoc[cloudwatch-dashboard Service Migration Guide]
+* link:./refarch2-guides/ecr-repos.adoc[ecr-repos Service Migration Guide]
+* link:./refarch2-guides/ecs-cluster.adoc[ecs-cluster Service Migration Guide]
+* link:./refarch2-guides/ecs-service-with-alb.adoc[ecs-service-with-alb Service Migration Guide]
+* link:./refarch2-guides/eks.adoc[EKS Service Migration Guide]
+* link:./refarch2-guides/iam-cross-account.adoc[iam-cross-account Service Migration Guide]
+* link:./refarch2-guides/iam-groups.adoc[iam-groups Service Migration Guide]
+* link:./refarch2-guides/iam-user-password-policy.adoc[iam-user-password-policy Service Migration Guide]
+* link:./refarch2-guides/jenkins.adoc[Jenkins Service Migration Guide]
+* link:./refarch2-guides/kms-master-key.adoc[kms-master-key Service Migration Guide]
+* link:./refarch2-guides/memcached.adoc[Memcached Service Migration Guide]
+* link:./refarch2-guides/openvpn-server.adoc[OpenVPN Server Service Migration Guide]
+* link:./refarch2-guides/rds.adoc[RDS Service migration Guide]
+* link:./refarch2-guides/redis.adoc[Redis Service Migration Guide]
+* link:./refarch2-guides/route53-private.adoc[Route 53 (private) Migration Guide]
+* link:./refarch2-guides/route53-public.adoc[Route 53 (public) Migration Guide]
+* link:./refarch2-guides/sns-topics.adoc[sns-topics Service Migration Guide]
+* link:./refarch2-guides/vpc-app.adoc[VPC (app) Migration Guide]
+* link:./refarch2-guides/vpc-mgmt.adoc[VPC (mgmt) Migration Guide]
```

Comment:
These links are all broken when I render the website in docker. I think it has to do with how we setup the jekyll plugin but not entirely sure.

Reply:
Links updated to point to new location defined in https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50